config/secrets.yml

```diff
@@ -19,9 +19,14 @@
 
 development:
   secret_key_base: 6945e91dea8be1b2f9e19074c708d6cc290635b6d66445ebcbe0a499252341fb3c30516a483be4fa6490bd2335ac54b16351b2f2550f48f730ab8c59d0e7f5cd
-
+  FACEBOOK_APP_ID: "232349067302447"
+  FACEBOOK_APP_SECRET: "1fdf6fdf346b4fbef416cc838ce04d0f"
+  GOOGLE_APP_ID: "896454387297-9v2hq4fk49c93jpqhu0ftuspo4kscu0b.apps.googleusercontent.com"
+  GOOGLE_APP_SECRET: "8BHJnU8Z5ioMFnQWzTf5_ZVU"
 test:
   secret_key_base: d1a275994420f0bceb58156551df2e38a7503860c23a5da7a4ac99f98f2a4e01dc19dcbe7ecfa83d0ce583873336061e4c02db7459adbd4296e7016e9d9d6c1c
+  FACEBOOK_APP_ID: "232349067302447"
```

Comment:
I think you don't need these properties for test env. Faker has module for omniauth.